docs/get-started.md

```diff
@@ -49,24 +49,24 @@ title: Get started
   ```html
   <!--
```

Comment:
it would be wise to [generate this entire snippet](https://jspm.org/docs/generator/stable/classes/Generator.html#addMappings) for each build using the jspm generator package, that way it would always be up to date and we wouldn't have to maintain it.